specs/whisk/beacon-chain.md

```diff
@@ -0,0 +1,379 @@
+### Diagram
+
+```python
```

Comment:
don't see why we need pytohn syntax here
```suggestion
```
```

Reply:
The markdown syntax is completely broken without ```python. Github thinks it's some sort of table because of the `|`.